src/main/java/seedu/address/logic/commands/AddMeetingCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LOCATION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TITLE;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Attendee;
+import seedu.address.model.meeting.Meeting;
+
+/**
+ * Adds a person to the address book.
+ */
+public class AddMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "addm";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a meeting to the address book. "
+            + "Parameters: "
+            + PREFIX_TITLE + "TITLE "
+            + PREFIX_LOCATION + "LOCATION "
+            + PREFIX_START + "START "
+            + PREFIX_END + "END "
+            + "[" + PREFIX_NAME + "ATTENDEES]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TITLE + "Meeting with friends "
+            + PREFIX_LOCATION + "Starbucks "
+            + PREFIX_START + "18.10.2023 1200 "
+            + PREFIX_END + "18.10.2023 1400 "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_NAME + "Dohn Joe";
+
+    public static final String MESSAGE_SUCCESS = "New meeting added: %1$s";
+    public static final String MESSAGE_DUPLICATE_MEETING = "This meeting already exists in the address book";
+    public static final String MESSAGE_ATTENDEE_NOT_FOUND = "Meeting attendee is not found in Persons list";
+
+    private final Meeting toAdd;
+
+    /**
+     * Creates an AddCommand to add the specified {@code Person}
+     */
+    public AddMeetingCommand(Meeting meeting) {
+        requireNonNull(meeting);
+        toAdd = meeting;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (model.hasMeeting(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_MEETING);
+        }
+        for (Attendee attendee : toAdd.getAttendees()) {
+            if (!model.hasName(attendee.getAttendeeName())) {
+                throw new CommandException(MESSAGE_ATTENDEE_NOT_FOUND);
+            }
+        }
```

Comment:
remove the functionality to add attendees from the add meetings command for now

Reply:
I removed the surface level functionality, the addm command will no longer put any names into the attendee list.